auditbeat/module/file_integrity/event_linux.go

```diff
@@ -147,6 +179,59 @@ func metadataFromFileModify(evt *ebpfevents.FileModify) (Metadata, error) {
 	return md, err
 }
 
+func newProcess(pid uint32, start uint64, comm string, euid, egid uint32) (Process, error) {
+	var (
+		p   Process
+		err error
+	)
+
+	t, err := sys.TimeFromNsSinceBoot(start)
+	if err != nil {
+		return p, err
+	}
+
+	p.EntityID, err = sys.EntityID(pid, t)
+	if err != nil {
+		return p, err
+	}
+	p.Name = comm
+	p.PID = pid
+
+	p.User.ID = strconv.FormatUint(uint64(euid), 10)
+	u, err := user.LookupId(p.User.ID)
```

Comment:
`LookupId` and `LookupGroupId` below could use the network and possibly take multiple seconds to return.  (For example if Linux host was using LDAP for users/groups and connectivity had a transient error). 

Would that cause any problems like missing events?

Reply:
these functions are also used in other enrichment paths, also here they are used in a "best effort" way. If the function execution takes multiple seconds I see there's a possibility to slow down the event queue (but that possibility already exists), but I don't think I'm introducing new problems which would cause missing events